src/robotFactory.js

```diff
@@ -1,10 +1,69 @@
 'use strict';
 
-class BaseRobot {}
+// - Lets create a few classes of robots.
+// |-- damn...I love classes))
+class BaseRobot {
+  constructor(name, weight, coord, chipVersion) {
+    this.name = name;
+    this.weight = weight;
+    this.chipVersion = chipVersion;
 
-class FlyingRobot {}
+    this.coords = {
+      x: coord.x ? coord.x : 0,
+      y: coord.y ? coord.y : 0,
+    };
+  }
 
-class DeliveryDrone {}
+  getInfo() {
+    return `Robot: ${this.name}, `
+      + `Chip version: ${this.chipVersion}, `
+      + `Weight: ${this.weight}`;
+  }
```

Comment:
u can use destructuring
```suggestion
    cons { name, chipVersion, weight } = this
    
    return `Robot: ${name}, Chip version: ${chipVersion}, Weight: ${weight}`;
  }
```